src/rt/sections_osx.d

```diff
@@ -23,6 +23,13 @@ import core.sys.osx.mach.getsect;
 import rt.deh, rt.minfo;
 import rt.util.container.array;
 
+version (D_LP64)
+    import rt.sections_osx64;
+else
+    import rt.sections_osx32;
+
+mixin SectionsOSX;
```

Comment:
Actually, Looking at my code for shared libraries, I would have to change the SectionGroup struct and the onAddImage function as well. And per Walter's response in the forums, shared libraries should be made available for 64bit only as well. So this eliminates/differentiates most of what's still being shared between the two files sections_osx32.d and sections_osx64.d.

So could we go with my last suggestion, and have one file for each platform, and replace the imports in sections.d with this?

    else version(OSX)
    {
        version(D_LP64)
            public import rt.sections_osx64
        else
            public import rt.sections_osx32
    }